test/snapshots/blueprint.test.ts.md

```diff
@@ -70,27 +70,42 @@ Generated by [AVA](https://avajs.dev).
                   code: {
                     javascript: {
                       request: 'await seam.foos.get({"foo_id":"8d7e0b3a-b889-49a7-9164-4b71a0506a33"})',
+                      request_syntax: 'javascript',
                       response: '{"foo_id":"8d7e0b3a-b889-49a7-9164-4b71a0506a33","name":"Best foo","nullable_property":null}',
+                      response_syntax: 'json',
                       title: 'JavaScript',
                     },
                     php: {
                       request: '$seam->foos->get(foo_id:"8d7e0b3a-b889-49a7-9164-4b71a0506a33")',
+                      request_syntax: 'php',
                       response: '{"foo_id":"8d7e0b3a-b889-49a7-9164-4b71a0506a33","name":"Best foo","nullable_property":null}',
+                      response_syntax: 'json',
                       title: 'PHP',
                     },
                     python: {
                       request: 'seam.foos.get(foo_id="8d7e0b3a-b889-49a7-9164-4b71a0506a33")',
+                      request_syntax: 'python',
                       response: 'Foo(foo_id="8d7e0b3a-b889-49a7-9164-4b71a0506a33", name="Best foo", nullable_property=None)',
+                      response_syntax: 'python',
                       title: 'Python',
                     },
                     ruby: {
                       request: 'seam.foos.get(foo_id: "8d7e0b3a-b889-49a7-9164-4b71a0506a33")',
+                      request_syntax: 'ruby',
                       response: `<Seam::Foo:0x00000␊
                       foo_id="8d7e0b3a-b889-49a7-9164-4b71a0506a33"␊
                       name="Best foo"␊
                       nullable_property=nil>`,
+                      response_syntax: 'ruby',
                       title: 'Ruby',
                     },
+                    seam_cli: {
+                      request: 'seam foos get --foo_id ""8d7e0b3a-b889-49a7-9164-4b71a0506a33""',
```

Comment:
Not sure what's going on with the double `""`.

Reply:
Fixed in https://github.com/seamapi/blueprint/pull/58/commits/4ae96fd18940c33f039d1a7d5c0f53cd989f3c02